generator.yaml

```diff
@@ -346,3 +346,10 @@ resources:
     fields:
       Name:
         is_primary_key: true
+  DBProxy:
+    exceptions:
+      terminal_codes:
+        - InvalidSubnet
+    fields:
+      DBProxyName:
```

Comment:
did you want to rename this to `Name` like the other resources, @brucegucode?

Reply:
Like i replied above, I think it's ok for now. previously we do the rename to avoid duplicate between spec and status, which is not an issue for DBProxy CRD here

Reply:
Sorry, I missed your response above. That's not why we do the renames :) We do the renames to simplify and "de-stutter" the field names. For consistency with the existing resources like `DBParameterGroup`, `DBSubnetGroup` and `DBClusterParameterGroup` (the ones that are identified by a "name" field versus an "Identifier" field...) I think it would be best to rename this one here.

Reply:
I agree with you to keep align with other resources